functions-and-arrays.js

```diff
@@ -0,0 +1,237 @@
+// Iteration #1: Find the maximum
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a;
+  } else if (a < b) {
+    return b;
+  } else {
+    return a;
+  }
+}

```

Comment:
Muy bien resuelto! pero una cosilla, si te fijas solo cambia el resultado para la b, y para el resto de casos siempre siempre siempre sera la a (te recomiendo una cosilla, los nombres de los parametros vamos a cambiarlos y poner algo que represente lo que contiene vale?)
Es decir que te quedaria algo asi:
```suggestion
function maxOfTwoNumbers(numberA, numberB) {
  if (numberA < numberB) {
    return numberB;
  } else {
    return numberA;
  }
}

```

Reply:
Peeeeerfesto!!!!

Muchas gracias!!!